From 611cc5cc134f710f94fc8987375259bd8af34604 Mon Sep 17 00:00:00 2001
From: Tobias Blomberg <git@sm0svx.shacknet.nu>
Date: Mon, 19 Jun 2017 22:04:20 +0200
Subject: [PATCH] Fix compilation problem in Async::AudioDeviceAlsa

- On newer compilers the compilation would fail on ambiguous call to abs
---
 src/async/audio/AsyncAudioDeviceAlsa.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/async/audio/AsyncAudioDeviceAlsa.cpp b/src/async/audio/AsyncAudioDeviceAlsa.cpp
index 17d255e8..15d760d5 100644
--- a/src/async/audio/AsyncAudioDeviceAlsa.cpp
+++ b/src/async/audio/AsyncAudioDeviceAlsa.cpp
@@ -548,7 +548,7 @@ bool AudioDeviceAlsa::initParams(snd_pcm_t *pcm_handle)
     return false;
   }
 
-  if (::abs(real_rate - sample_rate) > 100)
+  if (::abs(static_cast<int>(real_rate) - sample_rate) > 100)
   {
     cerr << "*** ERROR: The sample rate could not be set to "
          << sample_rate << "Hz for ALSA device \"" << dev_name << "\". "
